test/test-79-npm/ffi-napi/ffi-napi.js

```diff
@@ -0,0 +1,4 @@
+'use strict';
+
+const vosk = require('vosk');
```

Comment:
this test is not packaging the app, check https://github.com/vercel/pkg/tree/main/test/test-80-compression-node-opcua as an example. Remember to also add a gitignore for node_modules

Reply:
I'm seeing the `test/test-79-npm/main.js` file and getting confused because I don't know to what extent the testing utility scripts already do stuff.

It seems like commands such as [npm install](https://github.com/vercel/pkg/blob/main/test/test-79-npm/main.js#L232), running [pkg](https://github.com/vercel/pkg/blob/dd9de59c9fca2751bf5d22b57bd9b03d43e85e80/test/test-79-npm/main.js#L303), and even [vacuum](https://github.com/vercel/pkg/blob/dd9de59c9fca2751bf5d22b57bd9b03d43e85e80/test/test-79-npm/main.js#L420)('node_modules') are already done by this main script, so I'm wondering what should actually be done by the test.

In essence this particular test should simply consist on installing `vosk`, and running a main.js file that contains `require('vosk')`. If the shared library load doesn't work, the execution would fail. But I lack the knowledge to do this within the testing framework of *pkg*, and also I'm not sure if there are multiplatform considerations that need to be addresses too.

(I'll be out of town since tomorrow, so I'm afraid I won't be able to look into this in the following weeks)